docs/FAQ.md

```diff
@@ -16,31 +16,31 @@ group: FAQ
 
 **Is there an easy way to see the status and health of a cluster?**
 
->   Yes, Once Tachyon is installed, the UI is available at `http://master_ip:19999`
+>   Yes, once Tachyon is installed, the UI is available at `http://master_ip:19999`
 
 **If I add new nodes to the cluster, will tachyon re-balance / move the cached blocks to the
 newly added nodes in order to balance disk space utilization between the nodes?**
 
->   No, Currently re-balancing of data blocks in not available.
+>   No, currently re-balancing of data blocks is not available.
 
 **Is Tachyon master fault tolerant?**
 
 >   Yes. Please see this [page](http://tachyon-project.org/Fault-Tolerant-Tachyon-Cluster.html).
 
-**What is under FS and how many Underlayer FS are supported?**
+**What is UnderFS and how many UnderlayerFS are supported?**
```

Comment:
@apc999 So would the correction be "What is UnderFS and how many UnderFS are supported?"